### PR TITLE
Subcommands instead of flags for namespace command

### PR DESCRIPTION
**Pull request by awmuncy**
The ENS namespace comment should use subcommands instead of flags for "get" and "set" operations

### CODE REVIEW

**Review by dtbuchholz — APPROVED** [review]

File: src/commands/namespace.ts [hunk 1]
```diff
@@ -1,70 +1,88 @@
...
+      records,
+      mappings,
+    };
+
+    console.dir(response, { depth: null });
   }
-};
+}
+
+export const builder: CommandBuilder<{}, Options> = (yargs) =>
```
Comment (carsonfarmer) [inline src/commands/namespace.ts:62]: nice

File: src/commands/namespace.ts [hunk 2]
```diff
@@ -1,70 +1,88 @@
...
+      getHandler
+    )
+    .command(
+      "set <domain> [mappings..]",
+      "Set text records for a namespace",
+      (yargs) =>
+        yargs
+          .positional("domain", {
+            type: "string",
+            description: "Blah",
```
Comment (carsonfarmer) [inline src/commands/namespace.ts:81]: Maybe a more concrete description?

File: src/commands/namespace.ts [hunk 3]
```diff
@@ -1,70 +1,88 @@
...
+
+export const builder: CommandBuilder<{}, Options> = (yargs) =>
+  yargs
+    .command(
+      "get <record>",
+      "Pass in a record to find it's table name",
+      (yargs) =>
+        yargs.positional("record", {
+          type: "string",
+          description: "The target table name",
```
Comment (carsonfarmer) [inline src/commands/namespace.ts:70]: "The mapped name"?

**Review by carsonfarmer — APPROVED** [review]
Have a few nit comments in there around descriptions and error messages. Otherwise, this seems like a solid update/fix.

**Comment by carsonfarmer** [comment]
Might also be useful to pull in all those pending dev deps and runtime deps just so we can get all those updated?

**Comment by carsonfarmer** [comment]
Ooop, also looks like some tests failing. Any idea why the one test error has changed?

**Pull request merged**